suite-common/analytics/src/events/suite-native/types.ts

```diff
@@ -558,4 +561,23 @@ export type SuiteNativeAnalyticsEvent =
       }
     | {
           type: EventType.ReferralButtonPress;
+      }
+    | {
+          type: EventType.DemoAccountQuestionnaireDashboard;
+      }
+    | {
+          type: EventType.DemoAccountQuestionnaireStart;
+      }
+    | {
+          type: EventType.DemoAccountQuestionnaireQuestion;
+          payload: {
+              option: DemoAccountQuestionnaireQuestionOption;
+              question: DemoAccountQuestionnaireQuestion;
+          };
+      }
```

Comment:
What about something like: 
```suggestion
    | {
          type: EventType.DemoAccountQuestionnaireQuestion;
          payload: {
              option: QuestionAOptions;
              question: QuestionA;
          } |  {
              option: QuestionBOptions;
              question: QuestionB;
          };
      }
```
That would ensure that we will never send an answer that is meant for another question.

Reply:
I was also thinking about that but to be honest I'm not worried we will send answer for another question and even if we did, it's not such a big deal. I prefer simplicity here...

Reply:
Agree that it is not a big deal 👍